libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/xirr/PluggableXirr.tsx

```diff
@@ -85,6 +87,13 @@ export class PluggableXirr extends AbstractPluggableVisualization {
         return sanitizeFilters(newReferencePoint);
     };
 
+    public modifyInsightForDrillDown(
+        sourceVisualization: IInsight,
+        _drillDownContext: IDrillDownContext,
+    ): IInsight {
+        return sourceVisualization;
+    }
```

Comment:
same as headline; let's have the noop impl in abstractplugviz and it does not have to be here then.

Reply:
Fixed